### PR TITLE
Issue 59 - model refactoring

### PR DESCRIPTION
**Pull request by bearcheese**
PR related to #59

### CODE REVIEW

File: adele-clerk/src/test/java/com/microservicesteam/adele/clerk/boundary/web/ReservationControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+    @Test
+    public void reservePositionsShouldReturnWithReservationId() throws Exception {
+        when(reservationsService.reservePositions(any()))
+                .thenReturn(ReservationRequested.builder()
+                        .reservationId("someReservationId")
+                        .build());
+        String requestBody = "{\"positions\":[{\"eventId\":1,\"sectorId\":2,\"seatId\":3}]}";
+        System.out.println(requestBody);
```
Comment (bearcheese) [inline adele-clerk/src/test/java/com/microservicesteam/adele/clerk/boundary/web/ReservationControllerTest.java]: Should be removed!

File: adele-clerk/src/main/java/com/microservicesteam/adele/clerk/boundary/web/ReservationController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+@RestController
+@RequestMapping("reservations")
+@AllArgsConstructor
+public class ReservationController {
+
+    private final ReservationsService reservationsService;
+
+    @PostMapping
+    public ReservationResponse reservePositions(@RequestBody ReservationRequest reservationRequest) {
```
Comment (kelemena) [inline adele-clerk/src/main/java/com/microservicesteam/adele/clerk/boundary/web/ReservationController.java]: We should rename `position` to `ticketId`.
We should rename `reservePositions` to `reserveTickets`.
And also `ReservationRequest` should contains List<TicketId>.

File: adele-clerk/src/main/java/com/microservicesteam/adele/clerk/domain/TicketsService.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+public class TicketsService extends EventBasedService {
+    private final TicketRepository ticketRepository;
+
+    public TicketsService(EventBus eventBus,
+            TicketRepository ticketRepository) {
+        super(eventBus);
+        this.ticketRepository = ticketRepository;
+    }
+
+    public ImmutableList<Ticket> getTicketsStatusByEvent(long eventId) {
```
Comment (kelemena) [inline adele-clerk/src/main/java/com/microservicesteam/adele/clerk/domain/TicketsService.java]: @bearcheese I suggest to rename it to getTicketsByEvent

File: adele-clerk/src/main/java/com/microservicesteam/adele/clerk/domain/TicketsService.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        return ticketRepository.getTicketsStatusByEvent(eventId);
+    }
+
+    public ImmutableList<Ticket> getTicketsStatusByEventAndSector(long eventId, int sector) {
```
Comment (kelemena) [inline adele-clerk/src/main/java/com/microservicesteam/adele/clerk/domain/TicketsService.java]: @bearcheese I suggest to rename it to getTicketsByEventAndSector

**Pull request merged**